buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -441,43 +446,47 @@ def get_summary_table(identifier) {
         buildReleases.each { release ->
             // table cell
             def innerTable = "<table><tbody>"
+            def downstreamJobNames = buildFile.get_downstream_job_names(spec, release, identifier)
+            def downstreamBuilds = [:]
+            def pipelineName = ''
+            def build = null
+            def pipelineLink = '&nbsp;'
+            def pipelineRuntime = '&nbsp;'
 
             // check if this release is supported for this spec
             if (BUILD_SPECS.get(spec).contains(release)) {
-                def pipelineName = get_pipeline_name(spec, release)
-                def build = pipelineBuilds.get(pipelineName)
-                def downstreamJobNames = buildFile.get_downstream_job_names(spec, release, identifier)
+                pipelineName = get_pipeline_name(spec, release)
+                build = pipelineBuilds.get(pipelineName)
 
                 if (build) {
-                    def pipelineLink = buildFile.get_build_embedded_status_link(build)
-                    innerTable += "<tr style=\"text-align: right\"><td colspan=\"2\">${pipelineLink}</td><td>${build.getDurationString()}</td></tr>"
+                    pipelineLink = buildFile.get_build_embedded_status_link(build)
+                    downstreamBuilds.putAll(buildFile.get_downstream_builds(build, pipelineName, downstreamJobNames.values()))
+                    pipelineRuntime = build.getDurationString()
+                }
+            }
 
-                    // add pipeline's downstream builds
-                    def downstreamBuilds = buildFile.get_downstream_builds(build, pipelineName, downstreamJobNames.values())
+            innerTable += "<tr><td>&nbsp;</td><td style=\"text-align: right;\">${pipelineLink}</td><td style=\"text-align: right;\">${pipelineRuntime}</td></tr>"
 
-                    downstreamJobNames.each { label, jobName ->
-                        def downstreamBuild = downstreamBuilds.get(jobName)
-                        def link = ''
-                        def runtime = ''
+            // add pipeline's downstream builds
+            downstreamJobNames.each { label, jobName ->
+                def downstreamBuild = downstreamBuilds.get(jobName)
+                def link = '&nbsp;'
+                def runtime = '&nbsp;'
```

Comment:
Same comment about `RunTime` or `duration`.

Reply:
`duration` it is